Livecheckables/arx-libertatis.rb

```diff
@@ -0,0 +1,3 @@
+class ArxLibertatis
+  livecheck :regex => /^v?(\d+(?:\.\d+)+)b?$/
```

Comment:
```suggestion
  livecheck :regex => /^v?(\d+(?:\.\d+)+)$/
```

As in the other PR, just use the unmodified standard regex for Git tags. Adding `b?` here would potentially allow for beta versions in the future, which we want to avoid.

Reply:
Nice catch, I will remove it.